src/java/org/endlessos/FullScreen.java

```diff
@@ -0,0 +1,87 @@
+package org.endlessos;
```

Comment:
```suggestion
package org.learningequality;
```

Could we rename the file as well?